.github/workflows/publish.yml

```diff
@@ -0,0 +1,25 @@
+name: Publish to GitHub Pages
+on:
+  push:
+    branches: [docs]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+    - name: Install Node.js
+      uses: actions/setup-node@v2
+      with:
+        node-version: '16'
+    - name: Install Antora
+      run: npm i antora
+    - name: Generate Site
+      run: | 
+        cd docs
+        npx antora antora-playbook.yml
```

Comment:
Why is this called `antora-playbook`? Let's name it more meaningfully.

Reply:
It is common practice with antora in all of their documentation to call the file antora-playbook.yml, so it may be clearer to leave it like this and then people can research the purpose of the playbook. However it is possible to change the name, so we can do so if you think that it is best. Maybe to something like site-generator.yml?

Reply:
If it's common practice, that's fine. I thought it was just a name copied from  specific example.